src/js/components/RadioButtonGroup/RadioButtonGroup.js

```diff
@@ -101,7 +107,7 @@ const RadioButtonGroup = forwardRef(
         onLeft={focus ? onPrevious : undefined}
         onRight={focus ? onNext : undefined}
       >
-        <Box ref={ref} gap={gap} {...rest}>
+        <Box ref={ref} gap={radioButtonGap} {...rest}>
```

Comment:
```suggestion
        <Box ref={ref} {...theme.radioButtonGroup} {...rest}>
```
This change will enable all Box props and not just `gap`.

Reply:
changes made